docs/README.md

```diff
@@ -7,7 +7,7 @@ These instructions will get you a copy of the eclipse-ee4j.github.io/cdi/ websit
 ### Installation
 [Jekyll static site generator docs](https://jekyllrb.com/docs/).
 
-1. Install a full [Ruby development environment](https://jekyllrb.com/docs/installation/)
+1. Install a full [Ruby development environment](https://jekyllrb.com/docs/installation/) (Version 2.x is recommended, version 3.x seems to be incompatible)
```

Comment:
I just tried to follow these instructions locally and hit troubles due to having Ruby 2.5. In particular, the pinned version of Nokogiri seems to require at least Ruby 2.6. I installed Ruby 2.7 and everything worked fine. I didn't have the courage to try Ruby 3.x :-)

```suggestion
1. Install a full [Ruby development environment](https://jekyllrb.com/docs/installation/) (Version 2.7 is recommended, version 3.x seems to be incompatible)
```